Test fails from timeout on master

This test is causing a lot of PRs to have failing CI:

```
=================================== FAILURES ===================================
[223](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:224)
_______________________________ test_issue_8404 ________________________________
[224](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:225)
[gw3] linux -- Python 3.14.5 /opt/hostedtoolcache/Python/3.14.5/x64/bin/python
[225](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:226)

[226](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:227)
self = (6*re(exp(-2*I*pi/3)*polylog(2, exp(2*I*pi/3))) + 6*re(exp(2*I*pi/3)*polylog(2, exp(-2*I*pi/3))) + pi**2 + I*(6*im(exp...))))**2 + (6*re(exp(-2*I*pi/3)*polylog(2, exp(2*I*pi/3))) + 6*re(exp(2*I*pi/3)*polylog(2, exp(-2*I*pi/3))) + pi**2)**2)
[227](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:228)

[228](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:229)
    def getit(self):
[229](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:230)
        try:
[230](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:231)
>           return self._assumptions[fact]
[231](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:232)
                   ^^^^^^^^^^^^^^^^^^^^^^^
[232](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:233)
E           KeyError: 'negative'
[233](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:234)

[234](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:235)
sympy/core/assumptions.py:508: KeyError
[235](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:236)

[236](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:237)
During handling of the above exception, another exception occurred:
[237](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:238)

[238](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:239)
self = 6*re(exp(-2*I*pi/3)*polylog(2, exp(2*I*pi/3))) + 6*re(exp(2*I*pi/3)*polylog(2, exp(-2*I*pi/3))) + pi**2 + I*(6*im(exp(2*I*pi/3)*polylog(2, exp(-2*I*pi/3))) + 6*im(exp(-2*I*pi/3)*polylog(2, exp(2*I*pi/3))))
[239](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:240)

[240](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:241)
    def getit(self):
[241](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:242)
        try:
[242](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:243)
>           return self._assumptions[fact]
[243](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:244)
                   ^^^^^^^^^^^^^^^^^^^^^^^
[244](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:245)
E           KeyError: 'infinite'
[245](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:246)

[246](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:247)
sympy/core/assumptions.py:508: KeyError
[247](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:248)

[248](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:249)
During handling of the above exception, another exception occurred:
[249](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:250)

[500](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:501)
    return polylog_unitcircle(ctx, int(ctx.re(s)), z)
[501](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:502)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[502](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:503)
/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/mpmath/functions/zeta.py:425: in polylog_unitcircle
[503](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:504)
    if term and abs(term) < tol:
[504](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:505)
                ^^^^^^^^^
[505](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:506)
/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/mpmath/ctx_mp_python.py:574: in __abs__
[506](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:507)
    v._mpf_ = mpc_abs(self._mpc_, prec, rounding)
[507](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:508)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[508](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:509)
/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/mpmath/libmp/libmpc.py:111: in mpc_abs
[509](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:510)
    return mpf_hypot(a, b, prec, rnd)
[510](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:511)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
[511](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:512)
/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/mpmath/libmp/libmpf.py:1821: in mpf_hypot
[512](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:513)
    return mpf_sqrt(hypot2, prec, rnd)
[513](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:514)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
[514](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:515)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[515](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:516)

[516](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:517)
s = (0, 1533296456792865, -84, 51), prec = 51, rnd = 'n'
[517](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:518)

[518](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:519)
>   def mpf_sqrt(s, prec, rnd=round_fast):
[519](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:520)
E   Failed: Timeout (>10.0s) from pytest-timeout.
[520](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:521)

[521](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:522)
/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/mpmath/libmp/libmpf.py:1787: Failed
[522](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:523)
----------------------------- Captured stdout call -----------------------------
[523](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:524)
~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139655175468736) ~~~~~~~~~~~~~~~~~~~~~
[524](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:525)
  File "/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/execnet/gateway_base.py", line 411, in _perform_spawn
[525](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:526)
    reply.run()
[526](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:527)
  File "/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/execnet/gateway_base.py", line 341, in run
[527](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:528)
    self._result = func(*args, **kwargs)
[528](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:529)
  File "/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/execnet/gateway_base.py", line 1160, in _thread_receiver
[529](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:530)
    msg = Message.from_io(io)
[530](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:531)
  File "/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/execnet/gateway_base.py", line 567, in from_io
[531](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:532)
    header = io.read(9)  # type 1, channel 4, payload 4
[532](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:533)
  File "/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/site-packages/execnet/gateway_base.py", line 534, in read
[533](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:534)
    data = self._read(numbytes - len(buf))
[534](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:535)
                                DO *NOT* COMMIT!                                
[535](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:536)
============================= slowest 50 durations =============================
[536](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:537)
10.00s call     sympy/utilities/tests/test_wester.py::test_W19
[537](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:538)
10.00s call     sympy/functions/special/tests/test_zeta_functions.py::test_issue_8404
[538](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:539)
8.97s call     sympy/assumptions/tests/test_query.py::test_prime
[539](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:540)
8.83s call     sympy/combinatorics/tests/test_coset_table.py::test_look_ahead
[540](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:541)
8.82s call     sympy/series/tests/test_limits.py::test_issue_17431
[541](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:542)
8.46s call     sympy/polys/series/tests/test_hypothesis.py::test_rs_series_zero
[542](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:543)
8.43s call     sympy/polys/series/tests/test_hypothesis.py::test_global_series_one
[543](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:544)
8.29s call     sympy/integrals/tests/test_manual.py::test_issue_6746
[544](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:545)
8.11s call     sympy/physics/control/tests/test_lti.py::test_StateSpace_dsolve
[545](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:546)
8.01s call     sympy/integrals/tests/test_meijerint.py::test_messy
[546](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:547)
7.83s call     sympy/solvers/ode/tests/test_riccati.py::test_solve_riccati
[547](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:548)
7.79s call     sympy/physics/hep/tests/test_gamma_matrices.py::test_gamma_matrix_trace
[548](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:549)
7.79s call     sympy/holonomic/tests/test_holonomic.py::test_to_meijerg
[549](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:550)
7.73s call     sympy/matrices/tests/test_eigen.py::test_jordan_form
[550](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:551)
7.64s call     sympy/solvers/ode/tests/test_systems.py::test_sysode_linear_neq_order1_type3
[551](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:552)
7.54s call     sympy/series/tests/test_limits.py::test_atan
[552](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:553)
7.21s call     sympy/series/tests/test_aseries.py::test_simple
[553](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:554)
7.19s call     sympy/integrals/tests/test_integrals.py::test_issue_21831
[554](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:555)
7.16s call     sympy/parsing/tests/test_mathematica.py::test_Mathematica_literal_regex
[555](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:556)
7.08s call     sympy/solvers/ode/tests/test_ode.py::test_classify_ode
[556](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:557)
7.07s call     sympy/integrals/tests/test_integrals.py::test_sqrt_quadratic
[557](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:558)
6.95s call     sympy/stats/tests/test_continuous_rv.py::test_multiple_normal
[558](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:559)
6.89s call     sympy/solvers/tests/test_solvers.py::test_issue_20747
[559](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:560)
6.86s call     sympy/solvers/ode/tests/test_single.py::test_2nd_2F1_hypergeometric_integral
[560](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:561)
6.85s call     sympy/stats/tests/test_finite_rv.py::test_robust_soliton
[561](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:562)
6.75s call     sympy/matrices/expressions/tests/test_matpow.py::test_as_explicit_symbol
[562](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:563)
6.74s call     sympy/functions/special/tests/test_bessel.py::test_bessely_series
[563](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:564)
6.67s call     sympy/concrete/tests/test_sums_products.py::test_geometric_sums
[564](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:565)
6.62s call     sympy/integrals/tests/test_integrals.py::test_issue_14078
[565](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:566)
6.61s call     sympy/physics/tests/test_secondquant.py::test_dummy_order_ambiguous
[566](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:567)
6.58s call     sympy/integrals/tests/test_integrals.py::test_issue_23718
[567](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:568)
6.56s call     sympy/polys/tests/test_polyroots.py::test_roots_quartic
[568](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:569)
6.55s call     sympy/core/tests/test_function.py::test_issue_17382
[569](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:570)
6.52s call     sympy/physics/continuum_mechanics/tests/test_beam.py::test_max_bmoment
[570](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:571)
6.39s call     sympy/simplify/tests/test_trigsimp.py::test_trigsimp_groebner
[571](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:572)
6.25s call     sympy/series/tests/test_limits.py::test_issue_14556
[572](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:573)
6.17s call     sympy/solvers/ode/tests/test_systems.py::test_nonlinear_2eq_order1
[573](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:574)
6.17s call     sympy/functions/special/tests/test_zeta_functions.py::test_lerchphi_expansion
[574](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:575)
6.13s call     sympy/series/tests/test_limits.py::test_issue_15202
[575](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:576)
6.10s call     sympy/physics/control/tests/test_lti.py::test_StateSpace_series
[576](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:577)
6.09s call     sympy/physics/control/tests/test_lti.py::test_DiscreteStateSpace_series
[577](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:578)
6.05s call     sympy/simplify/tests/test_hyperexpand.py::test_meijerg_shift_operators
[578](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:579)
6.03s call     sympy/solvers/ode/tests/test_ode.py::test_issue_22523
[579](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:580)
6.02s call     sympy/functions/special/tests/test_bessel.py::test_besselk_series
[580](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:581)
5.99s call     sympy/integrals/tests/test_transforms.py::test_fourier_transform
[581](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:582)
5.89s call     sympy/solvers/ode/tests/test_single.py::test_2nd_nonlinear_autonomous_conserved_integral
[582](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:583)
5.89s call     sympy/ntheory/tests/test_residue.py::test_binomial_p_pow
[583](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:584)
5.87s call     sympy/concrete/tests/test_sums_products.py::test_issue_14640
[584](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:585)
5.83s call     sympy/integrals/tests/test_manual.py::test_issue_8520
[585](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:586)
5.82s call     sympy/functions/elementary/tests/test_hyperbolic.py::test_acsch_nseries
[586](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:587)
=========================== short test summary info ============================
[587](https://github.com/sympy/sympy/actions/runs/27083951588/job/79934809430?pr=29847#step:6:588)
FAILED sympy/functions/special/tests/test_zeta_functions.py::test_issue_8404 - Failed: Timeout (>10.0s) from pytest-timeout.
```

--- a/sympy/functions/special/tests/test_zeta_functions.py
+++ b/sympy/functions/special/tests/test_zeta_functions.py
@@ -11,7 +11,7 @@ from sympy.functions.special.zeta_functions import (dirichlet_eta, lerchphi, pol
 from sympy.series.order import O
 from sympy.core.function import ArgumentIndexError
 from sympy.functions.combinatorial.numbers import bernoulli, factorial, genocchi, harmonic
-from sympy.testing.pytest import raises
+from sympy.testing.pytest import raises, slow
 from sympy.core.random import (test_derivative_numerically as td,
                       random_complex_number as randcplx, verify_numerically)
 
@@ -182,6 +182,7 @@ def test_polylog_series():
         - sqrt(3)*z**3/9 + z**4/8 + O(z**5)
 
 
+@slow
 def test_issue_8404():
     i = Symbol('i', integer=True)
     assert Abs(Sum(1/(3*i + 1)**2, (i, 0, S.Infinity)).doit().n(4)

--- a/sympy/utilities/_compilation/tests/test_compilation.py
+++ b/sympy/utilities/_compilation/tests/test_compilation.py
@@ -4,7 +4,7 @@ import os
 import subprocess
 import tempfile
 from sympy.external import import_module
-from sympy.testing.pytest import skip, skip_under_pyodide
+from sympy.testing.pytest import skip, skip_under_pyodide, slow
 
 from sympy.utilities._compilation.compilation import compile_link_import_py_ext, compile_link_import_strings, compile_sources, get_abspath
 
@@ -66,6 +66,7 @@ def test_compile_link_import_strings():
             shutil.rmtree(info['build_dir'])
 
 
+@slow
 @skip_under_pyodide("Emscripten does not support subprocesses")
 def test_compile_sources():
     tmpdir = tempfile.mkdtemp()

